src/lib.rs

```diff
@@ -226,6 +247,65 @@ pub fn read_accessions(file: &PathBuf) -> Vec<String> {
         .collect()
 }
 
+pub fn print_csv(runs: Vec<Run>) -> Result<(), std::io::Error> {
```

Comment:
This is the long format, the other one is wide format: https://www.statology.org/long-vs-wide-data/#:~:text=A%20dataset%20can%20be%20written,repeat%20in%20the%20first%20column.&text=Notice%20that%20in%20the%20wide,the%20first%20column%20is%20unique.